### ISSUE TITLE
Roborock S6 MaxV error mse: camera status denied preview, unexpected EOF

### ISSUE THREAD

**Issue by SergiiTrybushnyi** [AlexxIT/go2rtc#393]
Hi all, 

I am trying to acces Roborock S6 MaxV camera, however getting error while accessing stream: 

`mse: camera status denied preview, unexpected EOF`

my config file:
`streams:
  camera1: roborock://mqtt-eu-5.roborock.com:8883?u=7XD3ntHm0pY0FVjNx647qM&s=xxx&k=xxx&did=xxx&key=xxxx&pin=`

Could you please advise what is wrong in my set up?

**Comment by AlexxIT** [comment]
Maybe you have pin code for connection?

**Comment by SergiiTrybushnyi** [comment]
Nope, don't have any pins, connected through wifi with roborock app and that's it
(i might get question wrong, but never used any pins for vacuum)

i resolved camera status denied preview, 

Still have "unexpected EOF" error

**Comment by peteh** [comment]
Do you have to enter pattern on your phone to access camera?

**Comment by SergiiTrybushnyi** [comment]
> Do you have to enter pattern on your phone to access camera?

Yes, I have to "draw" as on Android phones. Is this meant under "pin" ?

**Comment by peteh** [comment]
Yes, that worked for me. Imagine the pattern being a phone keyboard and the numbers you swipe through are the order of the pin numbers. 

You also have make sure the robot is not docked to see the video feed. In docked mode the video feed cannot be started.
